src/index.html

```diff
@@ -10,8 +12,155 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
+
+      <div class="container">
```

Comment:
use fieldsets instead, fix it everywhere
```suggestion
      <fieldset class="container">
```